src-ghc/Pact/ApiReq.hs

```diff
@@ -84,7 +83,7 @@ apiReq fp local = do
   if local then
     BSL.putStrLn $ encode exec
     else
-    BSL.putStrLn $ encode $ SubmitBatch [exec]
+    BSL.putStrLn $ encode $ SubmitBatch $ exec :| []
```

Comment:
this is just `pure`.

Reply:
Yes, and while I've used `pure` like that before, I figured using the constructor explicitly would be clearer for others.

Reply:
It would be better if NEL had `singleton` for consistency with other Data.Foo containers ... oh well